File: crates/deslop/tests/fixtures/python-issue-103-helper-call-sites/control_clone_b.py
# The pasted copy of the control clone. The banner differs so the two
# files are not byte-identical; `settle_ledger` itself is untouched.


def settle_ledger(entries, floor, penalty):
    balance = 0
    for entry in entries:
        if entry > floor:
            balance += entry * 3 - penalty
        else:
            balance += entry // 2
    return balance

File: crates/deslop/tests/fixtures/ts-issue-285-diagnostic-scenarios/control_clone_b.ts
// The pasted copy of the control clone. Only this banner differs.

export function settleLedger(entries: number[], floor: number, penalty: number): number {
  let balance = 0;
  for (const entry of entries) {
    if (entry > floor) {
      balance += entry * 3 - penalty;
    } else {
      balance += Math.floor(entry / 2);
    }
  }
  return balance;
}
